simple-search-engine-maven/src/main/java/searchengine/util/SearchEngineUtils.java

```diff
@@ -0,0 +1,121 @@
+package searchengine.util;
+
+import searchengine.model.SearchEngineContext;
+import searchengine.model.SearchStrategy;
+import searchengine.model.SearchDataset;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+/**
+ * Utility class for Search Engine
+ */
+public class SearchEngineUtils {
+
+    public static final String MENU = """
+                            
+            === Menu ===
+            1. Find a person
+            2. Print all people
+            0. Exit
+            """;
+
+    private SearchEngineUtils() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    /**
+     * Indexing data in {@link SearchDataset}
+     *
+     * @param lines from {@link SearchDataset} to index
+     * @return {@link Map} with all indexed data
+     */
+    public static Map<String, Set<Integer>> invertIndexes(List<String> lines) {
```

Comment:
This function belongs more to ``SearchDataset`` as private pre-processing function or as static initializer ,
something like 
```java
public class SearchDataset {
    private List<String> lines; 
    private Map<String, Set<Integer>> invertedIndexes;

    public SearchDataset(List<String> lines) {
        this.lines = lines;
        this.map  = this.invertIndex(lines)
    }
   
     private Map<String, Set<Integer>> invertIndexes(List<String> lines) { .... }
  
}
```